postgis/postgis_before_upgrade.sql

```diff
@@ -173,3 +173,9 @@ BEGIN
 END;
 $$;
 
+-- FUNCTION ST_AsGeoJson added `id_column` optional argument in 3.5.0.
+SELECT _postgis_drop_function_by_identity
```

Comment:
For the record:  by adding a new parameter we actually created a NEW SIGNATURE (rather than changing the identity of an existing signature). So we'd have needed a different function (drop by signature) BUT the simplest way to deal with upgrades is the use a `Replaces` comment, like it was done to fix the bug which resulted by this misues of _postgis_drop_function_by_identity, see

https://trac.osgeo.org/postgis/ticket/5623#comment:5

https://git.osgeo.org/gitea/postgis/postgis/commit/a6f4fa8282d3fef7e298f66cb7db96674486f028